tests/components/light.test.js

```diff
@@ -38,6 +38,12 @@ suite('light', function () {
     });
   });
 
+  test('can update parameters on updated light', function () {
```

Comment:
The only reason I didn't do that is because at that time, it's a directional light, rather than an ambient light.